src/app/hardfork_test/src/internal/hardfork/config_validation.go

```diff
@@ -0,0 +1,193 @@
+package hardfork
```

Comment:
these json validation is very verbose. We might want to consider simplifying them.